### PR TITLE
Update project profile in Home Unite Us page - #3929

### LINKED ISSUE #3929: Update Project Profile: Home Unite Us
### Overview
As developers, we need to ensure all project details are up to date on the HackforLA website. For this issue, we will update the project information for Home Unite Us.

### Action Items
- [x] Observe the existing syntax of the [front matter block](https://jekyllrb.com/docs/front-matter/) in the `_projects/home-unite-us.md` file.
- [x] Update the `techonologies` variable 
    - [x] Remove: auth0
    - [x] Add: Amazon Cognito
- [x] Remove the following person(s) from the `leadership` variable
    - [x] Kristelle Gumaru
- [x] Add the following persons to the `leadership` variable
    - [x] Name: Samuel Kowitch
GitHub Handle: KowDesign
Role: UX/UI Designer
Slack Member ID: U04GYSFB98X
    - [x] Name: Muyin Zheng
GitHub Handle: IndigoYinZ
Role: UX/UI Designer
Slack Member ID: U04HAEEKT29
    - [x] Name: Emily Eldar
GitHub Handle: emeldar
Role: Developer
Slack Member ID: U04BEDM1B2L
- [x] Check the Home Unite Us details page on your local development environment to verify the changes 
- [x] Include a screenshot of your successful changes with your pull request

### Instructions
- For the github link, append the team member's github handle to https://github.com/
- For the picture link, append the team member's github handle to https://avatars.githubusercontent.com/
- For the slack link, append the team member's slack ID to https://hackforla.slack.com/team/

### Resources
- The Project Markdown File: https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md
- [HFLA Wiki on Project Markdown Files](https://github.com/hackforla/website/wiki/project.md-file-template)
- [Jekyll Docs on front matter](https://jekyllrb.com/docs/front-matter/)

### PR DESCRIPTION
**Pull request by perlaroyerc**
Fixes #3929

### What changes did you make and why did you make them?

  - Removed "auth0" and added "Amazon Cognito" to the "technologies" variable.
  - Added three new team members to the "leadership" variable, including their name, role, slack, GitHub, and picture.
  - Removed one team member. 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screenshot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screenshot 2023-02-17 at 1 44 17 PM

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Screenshot 2023-02-17 at 1 50 47 PM

</details>

### CODE REVIEW

**Comment by Thinking-Panda** [comment]
Availability: Weekdays 10 AM - 3 PM
ETA: 10 PM 2/17/23

**Review by Thinking-Panda — COMMENTED** [review]
Hi @perlaroyerc 
The production website has technologies listed as React, NoSQL, Flask, Auth0 / OIDC / OAuth2, MongoDB. 
The issue mentions to remove auth0 and add Amazon Cognito. Should the change look like 
Option 1: React, NoSQL, Flask, Amazon Cognito / OIDC / OAuth2, MongoDB
Or
Option 2: React, NoSQL, Flask, OIDC / OAuth2, MongoDB, Amazon Cognito
Please confirm with @brycelednar

**Comment by randelbrot** [comment]
Hi @Thinking-Panda and @perlaroyerc , 

Option 1 looks good - thanks!

Bryce

**Review by christinaor — COMMENTED** [review]
I can see all the updates on the Home Unite Us project page in the Technologies section under Project Overview and in Current Project Team via Docker. Confirming that the front matter is formatted correctly in the markdown file as well. Awesome work there, Perla!

I'm not sure which issue the update in the website markdown is referring to or if I'm missing it in the referenced issue, but will act on this as soon as this part is clarified.

**Comment by Thinking-Panda** [comment]
Changes to home_unite_us.md are approved.

There are also changes applied to website.md that is not mentioned in the related issue.

**Comment by jdingeman** [comment]
Hi @perlaroyerc, I'm not sure if you may have accidentally worked on an old branch, but you will need to remove the changes made to website.md before we can merge this issue, as it has conflicts.

File: _projects/website.md [hunk 1]
```diff
@@ -30,18 +30,6 @@ leadership:
       slack: 'https://hackforla.slack.com/team/U03VDRGD34P'
       github: 'https://github.com/bishrfaisal'
     picture: https://avatars.githubusercontent.com/bishrfaisal
```
Comment (jdingeman) [inline _projects/website.md:32]: Hi @perlaroyerc, I'm not sure if you may have accidentally worked on an old branch, but you will need to remove the changes made to website.md before we can merge this issue, as it has conflicts.

**Review by jdingeman — CHANGES_REQUESTED** [review]

File: _projects/website.md [hunk 2]
```diff
@@ -30,18 +30,6 @@ leadership:
...
-  - name: Stephanie Samuels
```
Comment (jdingeman) [inline _projects/website.md:33]: Hi @perlaroyerc, I'm not sure if you may have accidentally worked on an old branch, but you will need to remove the changes made to website.md before we can merge this issue, as it has conflicts.

**Review by jdingeman — CHANGES_REQUESTED** [review]

File: _projects/website.md (hunk 2 above)
Comment (perlaroyerc) [reply _projects/website.md:33]: Hi @jdingeman! My apologies, I will make sure to switch to the correct branch and remove any changes that are causing conflicts as soon as possible.

**Pull request closed without merging**